lib/matplotlib/tests/test_figure.py

```diff
@@ -234,3 +237,39 @@ def test_figaspect():
     assert h / w == 0.5
     w, h = plt.figaspect(np.zeros((2, 2)))
     assert h / w == 1
+
+
+@pytest.mark.parametrize('which', [None, 'both', 'major', 'minor'])
```

Comment:
At the risk of sounding pedantic I'll note that this test belongs to test_dates.py.